app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -56,4 +56,18 @@ def has_proxy?
   def requires_storage_for_scan?
     false
   end
+
+  def require_snapshot_for_scan?
```

Comment:
@roliveri no, when you check to see if you need to add a snapshot for non-managed disks, it will always return false, won't it?